### PR TITLE
fix: gas fee base unit

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

We are using `networkFee` as `networkFeeBaseUnit` in some places, and `networkFeeCryptoHuman` in others.

This PR properly types the difference by updating `DisplayFeeData` to reflect `web`'s  consumption of the gas fee (as crypto human balance) and ensures the values are set as such consistently across all chains.

This fixes the wildly high fee issue we currently have on `develop`.

**Outstanding issues before opening:**

- [x] Trade confirm fees from `lib` are `0`. https://github.com/shapeshift/lib/pull/1045 will fix this.
- [ ] Quote data gets stale and results in wrong or `0` gas fee

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

Touches the consumption and transformation of `networkFees`, medium risk, though fairly quick to determine if it's working as expected.

## Testing

Get quotes for `0x`, `Cowswap`, and `ThorSwap` and check values on both the quote screen and the trade confirm screen.

### Engineering

As above.

### Operations

As above.

## Screenshots (if applicable)

N/A

### CODE REVIEW

File: src/components/Trade/types.ts [hunk 1]
```diff
@@ -26,7 +26,10 @@ export type TradeAsset = {
   fiatAmount?: string
 }
 
-export type DisplayFeeData<C extends ChainId> = QuoteFeeData<C> & { tradeFeeSource: string }
+export type DisplayFeeData<C extends ChainId> = Omit<QuoteFeeData<C>, 'networkFee'> & {
+  tradeFeeSource: string
+  networkFeeCryptoHuman: string
```
Comment (0xdef1cafe) [inline src/components/Trade/types.ts:31]: eventually we prob want this field in lib but this gets us outta trouble

File: src/components/Trade/hooks/useSwapper/useSwapper.tsx [hunk 2]
```diff
@@ -612,31 +612,31 @@ export const useSwapper = () => {
     sellAsset: Asset
     tradeFeeSource: string
   }) => {
-    const feeBN = bnOrZero(trade?.feeData?.networkFee).dividedBy(
-      bn(10).exponentiatedBy(feeAsset.precision),
```
Comment (0xdef1cafe) [inline src/components/Trade/hooks/useSwapper/useSwapper.tsx:616]: 🤣

**Review by 0xdef1cafe — APPROVED** [review]
approved pending DM comments that we don't need a package bump

File: src/components/Trade/types.ts (hunk 1 above)
Comment (0xApotheosis) [reply src/components/Trade/types.ts:31]: Agree 👌

File: src/components/Trade/hooks/useSwapper/useSwapper.tsx (hunk 2 above)
Comment (0xApotheosis) [reply src/components/Trade/hooks/useSwapper/useSwapper.tsx:616]: Hand-rolled `fromBaseUnit` 😬

**Pull request merged**